crates/feedback/src/feedback.rs

```diff
@@ -20,43 +20,39 @@ actions!(
 );
 
 pub fn init(cx: &mut AppContext) {
-    // TODO: a way to combine these two into one?
-    cx.observe_new_views(feedback_modal::FeedbackModal::register)
-        .detach();
+    cx.observe_new_views(|workspace: &mut Workspace, cx| {
+        // First registration
```

Comment:
I don’t think we need this comment on the registration call (or the one below). They don’t provide any value.

Reply:
I've just removed